docs/proposals/scheduling/20220722-multi-hierarchy-elastic-quota-management.md

```diff
@@ -341,8 +342,8 @@ labels:
 - `quota.scheduling.koordinator.sh/parent-quota-name` is disposed by the user. It reflects the parent quota name. Default is root.
 - `quota.scheduling.koordinator.sh/shared-weight` is disposed by the user. It reflects the ability to share the "lent to" resource. Default equals to "max".
 - `quota.scheduling.koordinator.sh/allow-lent-resource` is disposed by the user. It reflects whether quota group allows lent unused "min" to others.
-
-Here is a example:
+- `quota.scheduling.koordinator.sh/namespaces` is disposed by the user in need. It reflects the pod in the namespaces are bound to the quota, namespaces of quota should be different. The namespaces' definition is list of string.
```

Comment:
I overlooked some details. If a user creates many ElasticQuotas, binds many namespaces, and overlaps, is it allowed? Or how do we solve this problem?

Reply:
fixed